dashboard/src/actions/repos.test.tsx

```diff
@@ -953,7 +1050,7 @@ describe("validateRepo", () => {
     ];
 
     const res = await store.dispatch(
-      repoActions.validateRepo("url", "helm", "auth", "", "cert", [], false),
+      repoActions.validateRepo("url", "helm", "", "auth", "", "cert", [], false),
```

Comment:
I see. I figured some day we might want to do some validation of description on the back end, so I added. If you feel it would be cleaner not to do it today, I will remove description from validate() calls.

Reply:
Great, thanks. Yes, I'd add it as an arg to `validateRepo` only when we're going to actually use it.

Reply:
ok